tests/filesystem.go

```diff
@@ -309,10 +309,9 @@ func generateUUID(t *testing.T) string {
 
 func createFilesForPartitions(t *testing.T, partitions []*types.Partition) {
 	for _, partition := range partitions {
-		if partition.Files == nil {
-			continue
-		}
 		createFilesFromSlice(t, partition.MountPath, partition.Files)
+		createDirectoriesFromSlice(t, partition.MountPath, partition.Directories)
```

Comment:
Given this is making a call to `Mkdir` directly which will fail if the directory already exists should this go before `createFilesFromSlice` which will create the parent directory for files via `MkdirAll`?

Reply:
Yup, it should. Moved.